tests/Resources/ti.ui.ios.documentviewer.test.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-Present by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* global OS_VERSION_MAJOR */
+/* eslint-env mocha */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+const should = require('./utilities/assertions');
+const utilities = require('./utilities/utilities');
+
+const isCI = Ti.App.Properties.getBool('isCI', false);
+
+describe.ios('Titanium.UI.iOS',  () => {
+	it('#createDocumentViewer()', () => {
+		should(Ti.UI.iOS.createDocumentViewer).be.a.Function();
+	});
+});
+
+describe.ios('Titanium.UI.iOS.DocumentViewer', () => {
+
+	let documentViewer;
+
+	beforeEach(() => {
+		documentViewer = Ti.UI.iOS.createDocumentViewer({
+			url: 'example.html'
+		});
+	});
+
+	afterEach(done => {
```

Comment:
No need to pass in a callback function to this hook or call it, it runs sync.
```suggestion
	afterEach(() => {
```

In fact, I don't think you even need this hook since you're not doing any work on the proxy like calling a `release` method - so assigning null shouldn't really help/matter.